ush/detect_machine.sh

```diff
@@ -26,6 +26,8 @@ case $(hostname -f) in
   gaea9.ncrc.gov)      MACHINE_ID=gaea ;; ### gaea9
   gaea1[0-6].ncrc.gov) MACHINE_ID=gaea ;; ### gaea10-16
 
+  gaea5[1-8].ncrc.gov) MACHINE_ID=gaea ;; ### gaea51-58
+
```

Comment:
```suggestion
gaea5[1-8].ncrc.gov) MACHINE_ID=gaea ;; ### gaea51-58

```
I think it is ok to remove the lines 24-27 for old Gaea